docker/airflow/compose.yaml

```diff
@@ -0,0 +1,41 @@
+
+version: '3.4'
+
+services:
+  airflow-initdb:
+    build: ./airflow/
+    env_file:
+      - ../.env
+    entrypoint: "airflow db init"
+
+  airflow:
+    build: ./airflow/
+    env_file:
+      - ../.env
+    restart: unless-stopped
+    volumes:
+      - ./airflow/files/dags:/usr/local/airflow/dags
+    ports:
+      - "8080:8080"
```

Comment:
Instead of exposing this port, I suggest configuring the host NGINX webserver to point to this local address.
the subdomain airflow.epigraphhub.org is already available.

Reply:
not sure if I am following you ... normally I would just point a subdomain to a specif port (in the nginx config file).
could you give me more details or an example about that?

Reply:
That's it.  I added the subdomain to the DNS configuration. Now what's left is to update nginx.conf with something like this:
```conf
server {
        listen 80;
        server_name airflow.epigraphhub.org;

        location / {
                proxy_pass http://localhost:8080;
                proxy_set_header Host $host;
                proxy_set_header X-Real-IP $remote_addr;
                proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
                proxy_connect_timeout 150;
                proxy_send_timeout 100;
                proxy_read_timeout 100;
                proxy_buffers 4 32k;
                client_max_body_size 8m;
                client_body_buffer_size 128k;
        }
```

you may have to re-run certbot as well, not sure.

Reply:
oh I see .. sure .. so I would need to keep exposing the port in order to have it reachable from nginx.

Reply:
Yes, I gave the wrong impression. All I intended to say was that the DNS configuration for the airflow subdomain was ready to use.